Jenkinsfile

```diff
@@ -51,110 +103,92 @@ def gitMerge(String primaryBranch) {
   }
 }
 
+def props
+
 timestamps {
-  // TinyMCE builds need more CPU and RAM (especially eslint)
-  // NOTE: Ensure not to go over 7.5 CPU/RAM due to EC2 node sizes and the jnlp container requirements
-  tinyPods.nodeBrowser([
+  tinyPods.node([
     resourceRequestCpu: '6',
     resourceRequestMemory: '4Gi',
     resourceLimitCpu: '7.5',
     resourceLimitMemory: '4Gi'
   ]) {
-    def props = readProperties(file: 'build.properties')
-
+    props = readProperties(file: 'build.properties')
     String primaryBranch = props.primaryBranch
     assert primaryBranch != null && primaryBranch != ""
-    def runAllTests = env.BRANCH_NAME == primaryBranch
 
-    stage("Merge") {
+    stage('Merge') {
       // cancel build if primary branch doesn't merge cleanly
       gitMerge(primaryBranch)
     }
 
-    def platforms = [
-      [ os: "windows", browser: "chrome" ],
-      [ os: "windows", browser: "firefox" ],
-      [ os: "windows", browser: "MicrosoftEdge" ],
-      [ os: "macos", browser: "safari" ],
-      [ os: "macos", browser: "chrome" ],
-      [ os: "macos", browser: "firefox" ]
-    ]
-
-    def cleanAndInstall = {
-      echo "Installing tools"
-      exec("git clean -fdx modules scratch js dist")
+    stage('Install') {
       yarnInstall()
     }
 
-    def processes = [:]
-
-    // Browser tests
-    for (int i = 0; i < platforms.size(); i++) {
-      def platform = platforms.get(i)
-
-      def buckets = platform.buckets ?: 1
-      for (int bucket = 1; bucket <= buckets; bucket++) {
-        def suffix = buckets == 1 ? "" : "-" + bucket
-
-        // closure variable - don't inline
-        def c_bucket = bucket
-
-        def name = "${platform.os}-${platform.browser}${suffix}"
-
-        processes[name] = {
-          stage(name) {
-            node("bedrock-${platform.os}") {
-              echo("Bedrock tests for ${name}")
-
-              echo("Checking out code on build node: $NODE_NAME")
-              checkout(scm)
-
-              // windows tends to not have username or email set
-              tinyGit.addAuthorConfig()
-              gitMerge(primaryBranch)
+    stage('Type check') {
+      withEnv(["NODE_OPTIONS=--max-old-space-size=1936"]) {
+        exec("yarn ci-all-seq")
+      }
+    }
 
-              cleanAndInstall()
-              exec("yarn ci")
+    stage('Moxiedoc check') {
+      sh 'yarn tinymce-grunt shell:moxiedoc'
+    }
+  }
+  //
+
+  def platforms = [
+    [ browser: 'chrome', provider: 'aws', buckets: 3 ],
+    [ browser: 'edge', provider: 'aws', buckets: 3 ],
+    [ browser: 'firefox', provider: 'aws', buckets: 3 ],
+    [ browser: 'safari', provider: 'lambdatest', buckets: 2 ]
```

Comment:
We still need to test chrome on mac. Not sure about edge, but we need _something_.

Reply:
@jscasca ideally we want testing for:
* Chrome/Firefox/Edge on Windows w/ AWS Device Farm
* Chrome/Firefox/Safari on Mac (Sonoma) w/ LambdaTest

Reply:
Ok this is somewhat new to me. I didn't put this down the first time so I will update bedrock to receive a platform conf argument. This may take a bit longer

Reply:
I made a new PR in bderock to accept the platform name as argument here: https://github.com/tinymce/bedrock/pull/136 

We will need more concurrent tests in LambdaTest

Reply:
Still missing one - I missed it, but @tjdett said to include Firefox on mac